### ISSUE TITLE
Full width promo

### ISSUE THREAD

**Issue by angievanhegan** [MantaRayMedia/bgs#130]
# Done criteria
- [ ] Promo can be uploaded to Drupal
- [ ] Promo can be added to a page
- [ ] A link can be added to a promo

# Design
- [Example](https://invis.io/BSF98SH9Q)

**Comment by angievanhegan** [comment]
@cat-manta Am I just useless at cropping? I can't get the crop to match your design (or to make it look ok!) Thanks

**Comment by cat-manta** [comment]
@angievanhegan I had to do them differently when I exported the images so they had more height for different screens, here are the images with the ideal crop in the middle:
![promo-blue](https://user-images.githubusercontent.com/29398815/35504384-d5d81e9c-04da-11e8-8afa-3426f413bda6.jpg)
![promo-green](https://user-images.githubusercontent.com/29398815/35504385-d5eadcb2-04da-11e8-97fd-9a0fd64645d3.jpg)
![promo-pink](https://user-images.githubusercontent.com/29398815/35504387-d5fd25a2-04da-11e8-9e06-ac856907c2fd.jpg)
![promo-purple](https://user-images.githubusercontent.com/29398815/35504388-d60fcee6-04da-11e8-8bd2-82974fdc36d6.jpg)

**Comment by angievanhegan** [comment]
@pipe-manta The shape of the promo doesn't really match the shape of the crop in Drupal so I can't seem to make this work. Can you take a look please? Thanks

**Comment by ican-icant** [comment]
@StephMSlack the plan was always to have a groovy parallax promo http://bgs-dev.mrmdev.co.uk/parallax so we need to confirm your're happy with it, rather than the plain image version. The link takes you to a page with Parallax vs Centered

**Comment by StephMSlack** [comment]
@ican-icant happy with the parallax promo. Can we change the image for the promo ourselves or do you need to do that?

**Comment by ican-icant** [comment]
@StephMSlack Yes, you can, but we will need to colour the images for you. Otherwise it will take on the original colour of the image

**Comment by StephMSlack** [comment]
@cat-manta would you be able to send through the RGB/CMYK colours you are using to create the web promos?

**Comment by StephMSlack** [comment]
@angievanhegan - these are fine

**Comment by cat-manta** [comment]
@StephMSlack All the colours I have used are from the branding guidelines, are you having issues creating new images for the promos?

**Comment by StephMSlack** [comment]
Hi @cat-manta Doh, not thinking! My colleagues have access to photoshop & had a go at adding the colour wash to an image, but it didnt quite look the same :) The sample is on the bottom of my test page http://bgs-dev.mrmdev.co.uk/about/sandbox/stephs-test-page

**Comment by StephMSlack** [comment]
@angievanhegan this is good to go as the static version - I cant seem to get the parallax version working?

**Comment by cat-manta** [comment]
@StephMSlack I've put together a Photoshop file in your Google Drive Design folder (below) with each colour promo so you can just swap out the image easily :) I've added comments on how to use each layer in the layer names, so hopefully it's straightforward to use, but I'm happy if you or anyone else wants a quick zoom chat to go through the file.

https://drive.google.com/file/d/1Iq_Cu4K4eIeksJRvN3XKo9j_SjNkTMSX/view?usp=sharing

**Comment by angievanhegan** [comment]
@pipe-manta Can you show us how to make the parallax version work please? Thanks

**Comment by pipe-manta** [comment]
@angievanhegan It's the same functionality for all the promos, we just need to know if we can applied everywhere

**Comment by angievanhegan** [comment]
@pipe-manta So, do they have to be parallax or all normal?

**Comment by angievanhegan** [comment]
@StephMSlack Steve has confirmed that having the option to have parallax OR normal view is out of scope. You can choose the one you prefer, or we can provide a quote for giving you the option to do both. Thanks.

**Comment by StephMSlack** [comment]
@angievanhegan @ican-icant - Parallax version please. Could you let me know how much for the static option and I can see if it makes financial sense to have both?

**Comment by StephMSlack** [comment]
thanks @cat-manta

**Comment by angievanhegan** [comment]
@russ-manta please can you provide a quote as per Blocker 1 above

**Comment by russ-manta** [comment]
@angievanhegan  1 day for this

**Comment by angievanhegan** [comment]
@StephMSlack Please can you let us know if you want us to go ahead with this option for you to be able to choose parallax or not on a promo by promo basis. Thanks

**Comment by dilwarhussain** [comment]
@StephMSlack All promos are parallax on the site now.

**Comment by StephMSlack** [comment]
@angievanhegan @dilwarhussain these look great, thanks!
